app/assets/stylesheets/_catppuccin.scss

```
@@ -0,0 +1,138 @@
+/*!
+ * Catppuccin Macchiato palette for CipherSwarm
+ * https://github.com/catppuccin/catppuccin
+ *
+ * Dark-first theme with DarkViolet (Mauve) accents.
+ * Override Bootstrap SCSS variables BEFORE @import "bootstrap".
+ */
+
+// Catppuccin Macchiato — base palette
+$ctp-rosewater: #f4dbd6;
+$ctp-flamingo:  #f0c6c6;
```

Stylelint (17.4.0)

[error] Expected single space after ":" with a single-line value (scss/dollar-variable-colon-space-after)

(scss/dollar-variable-colon-space-after)